components/squad/wikis/ageofempires/squad_custom.lua

```diff
@@ -0,0 +1,75 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:Squad/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Squad = require('Module:Squad')
+local SquadRow = require('Module:Squad/Row')
+local SquadAutoRefs = require('Module:SquadAuto/References')
+local Table = require('Module:Table')
+
+local CustomSquad = {}
+
+function CustomSquad.run(frame)
+	error("AoE wiki doesn't support manual Squad Tables")
```

Comment:
Is this correct (there are no manual tables on AoE - ie all runs SquadAuto?)

Reply:
There are still some manual tables, but those are old template ones and not based on `Module:Squad`.
We are currently moving the data to Transfers and SquadAuto.

Reply:
How far along is the migration? Considering if we should switch over to the standardized Squad if it's a lot less

Reply:
```suggestion
	error("AoE wiki doesn't use {{Squad}}, rather it uses {{ActiveSquad}} etc")
```

Reply:
> How far along is the migration? Considering if we should switch over to the standardized Squad if it's a lot less

50/104 team pages in main space still on old templates.
The reason why i'd like to have AutoSquad functionality for the future is that our team pages aren't edited that often, and if rather new editors add transfers they often miss to edit a team's or player's page.